packages/block-editor/src/components/iframe/index.js

```diff
@@ -38,7 +38,14 @@ function bubbleEvent( event, Constructor, frame ) {
 		init[ key ] = event[ key ];
 	}
 
-	if ( event instanceof frame.ownerDocument.defaultView.MouseEvent ) {
+	// Check if the event is a MouseEvent generated within the iframe.
+	// If so, adjust the coordinates to be relative to the position of
+	// the iframe. This ensures that components such as Draggable
+	// receive coordinates relative to the window, instead of relative
+	// to the iframe. Without this, the Draggable event handler would
+	// result in components "jumping" position as soon as the user
+	// drags over the iframe.
+	if ( event instanceof frame.contentDocument.defaultView.MouseEvent ) {
```

Comment:
Do we have a way to validate this behavior with an e2e test or something?

Reply:
Good question! I wonder how far we could get with React Testing Library and mounting an `Iframe` and firing an event, rather than a full e2e test? I'll add it to my list, but will focus on a couple of other bug fixes for 6.4 in the immediate term. For now, I think the addition of the comment here should help us in the interim, so we know what to look for if we encounter a similar issue again.